scripts/olmo2_modal_openai.py

```diff
@@ -0,0 +1,251 @@
+# # Run OpenAI-compatible LLM inference with vLLM
+
+# This is a simple example of running an OLMo 2 model on Modal.com with an OpenAI API and vLLM.
+# It is based on Modal.com's own reference example for OpenAI with vLLM here:
+# https://github.com/modal-labs/modal-examples/blob/ed89980d7288cd35c57f23861ba1b1c8d198f68d/06_gpu_and_ml/llm-serving/vllm_inference.py
+
+import os
+import modal
```

Comment:
This is an extra dependency. Is there a place where you can explain what needs to be done to get this working?

Reply:
Added setup information to the README